mne/decoding/csp.py

```diff
@@ -129,8 +129,13 @@ def _fit(self, cov_a, cov_b):
         w_b = np.dot(np.dot(p, cov_b), p.T)
         # and solve it
         vals, vecs = linalg.eigh(w_a, w_b)
-        # sort vectors by discriminative power using eigen values
-        ind = np.argsort(np.maximum(vals, 1. / vals))[::-1]
+        # sort vectors by discriminative power using eigenvalues
+        ind = np.argsort(vals)[::-1]
+        vecs = vecs[:, ind]
+        # re-order (first, last, second, second last, third, ...)
+        ind = np.empty_like(ind)
+        ind[::2] = np.arange(0, np.ceil(len(ind)/2))
+        ind[1::2] = np.arange(len(ind) - 1, int(len(ind)/2) - 1, -1)
```

Comment:
Spaces around `/` are not required - PEP8 says that [operators with the lowest priorities should be surrounded by spaces](https://www.python.org/dev/peps/pep-0008/#other-recommendations), so I guess in this case the style is OK.